applications/nrf5340_audio/src/bluetooth/bt_stream/broadcast/broadcast_source.c

```diff
@@ -231,18 +230,10 @@ int broadcast_source_ext_adv_populate(uint8_t big_index,
 
 	sys_put_le16(BT_UUID_BROADCAST_AUDIO_VAL, ext_adv_data->brdcst_id_buf);
 
-	if (IS_ENABLED(CONFIG_BT_AUDIO_USE_BROADCAST_NAME_ALT)) {
```

Comment:
This might change the demo behavior from the broadcaster side, we might still need CONFIG_BT_AUDIO_USE_BROADCAST_NAME_ALT for setup different broadcaster naming.